packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/v1/responses_wrappers.py

```
@@ -197,7 +202,47 @@ def set_data_attributes(traced_response: TracedData, span: Span):
                 SpanAttributes.LLM_USAGE_CACHE_READ_INPUT_TOKENS,
                 usage.input_tokens_details.cached_tokens,
             )
-        # TODO: add reasoning tokens in output token details
+
+        # Usage - count of reasoning tokens
+        reasoning_tokens = None
+        # Try dict-style access first (common when response_dict is a dict)
+        try:
+            tokens_details = usage.get("output_tokens_details")
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)